inc/injectiontype.class.php

```diff
@@ -50,14 +50,14 @@ public static function getItemtypes($only_primary = false)
         $plugin = new Plugin();
         $values = [];
         foreach ($INJECTABLE_TYPES as $type => $from) {
-            $injectionclass = new $type();
+            $injectionclass = (object) new $type();
```

Comment:
It doesn't seem necessary
```suggestion
            $injectionclass = new $type();
```